index.js

```diff
@@ -17,6 +33,14 @@ module.exports = function Rivet(url, options) {
     )
   }
 
+  /**
+   *
+   * @param {Object} params
+   * @param {string} params.query
+   * @param {string[]} [params.fragments]
```

Comment:
Technically when we use this as consumers we're get a _warning_ not to use it. 😈  

`dependencies` is the preferred param, though I had previously advocated for _both_ being valid. 

I digress... 

I'm no JSDoc expert but is there's something we do for a "deprecated" param, that may apply here - that's all I wanted to point out. After looking into it a bit, it doesn't sounds like `@deprecated` and `@param` can be used together

Reply:
This is probably a non-issue - moreso just thinking out loud.

Reply:
I'm totally down for making this a breaking change and fully removing the `fragments` parameter since I agree that it looks like we can't mark a param as deprecated.